Docs/how_to_build_on_linux.md

```diff
@@ -1,136 +1,194 @@
-<h1>How to build CARLA on Linux</h1>
----
+<h1>Linux build</h1>
 
-!!! note
-    CARLA requires Ubuntu 16.04 or later.
 
-Install the build tools and dependencies
+* [Requirements](#requirements):  
```

Comment:
"Minimum to run CARLA" and "Needed to build the project" gives no information. Either say it specifically or erase the subparts.

In general, I'd leave the subparts of the index, but erase their explanation, as most of them are redundant.
If needed, remark them in bold in their sections?